tcp_ip:
    sub_port: "5555"
    req_port: "4444"
    zipcode: "00000"

udp_ip:
    port: "8080"
    send_group: "FV1"
    recv_group: "LV"
